lib/reek/smells/control_parameter.rb

```diff
@@ -42,6 +42,8 @@ module Smells
     # the source code.
     #
     # See {file:docs/Control-Parameter.md} for details.
+    #
+    # :reek:InstanceVariableAssumption
```

Comment:
I kind of agree but I believe it's more important to finish this PR and get this merged.
There are a couple of things we should refactor before new features anyway.